aggoracle/oracle.go

```diff
@@ -78,36 +82,23 @@ func (a *AggOracle) processLatestGER(ctx context.Context) error {
 
 	latestGER := latestL1InfoLeaf.GlobalExitRoot
 
-	isGERInjected, err := a.chainSender.IsGERInjected(latestGER)
-	if err != nil {
-		return fmt.Errorf("error checking if GER (%s) is already injected: %w", latestGER, err)
-	}
-
-	if isGERInjected {
-		a.logger.Debugf("GER (%s) is already injected", latestGER.Hex())
-		return nil
-	}
-
 	go func() {
-		if err := a.injectGER(ctx, latestGER); err != nil {
+		// Submit GER based on mode
+		var err error
+		if a.enableAggOracleCommittee {
```

Comment:
We introduce this check in the `chainSender`:

```go
mode := DirectInjectionMode
if enableAggOracleCommittee && aggOracleCommitteeAddr != aggkitcommon.ZeroAddress {
    mode = AggOracleCommitteeMode
}
```

I don't think it is enough to say `if a.enableAggOracleCommittee` here. We might be in the inject mode.

I propose having a function called `ProcessGER` on the `chainSender`, which will based on its mode either call `ProposeGER` or `InjectGER`. I don't think oracle should know which mode on the sender is enabled.